src/index.html

```diff
@@ -5,7 +5,9 @@
   <meta charset="utf-8" />
   <title>GreenCity</title>
   <base href="/" />
-
+  <script type="application/javascript">
+    var global = window;
+  </script>
```

Comment:
![image](https://user-images.githubusercontent.com/6242270/89978504-58114580-dc76-11ea-92be-b3711e243e0c.png)

Go find another way instead of changing `index.html`.